Cannot find module 'i18next' or its corresponding type declarations.

Moving the i18next from dependencies to peerDependencies causes it to not be installed which causes the types to be missing.

Recommendation: duplicate all peerDependencies in devDependencies.

(remove .parcel-cache to reproduce)
```
packages/translations [main●] » yarn build
yarn run v1.22.17
$ yarn build:clean && parcel build
$ rimraf "./dist"

@parcel/transformer-typescript-types: Cannot find module 'i18next' or its corresponding type declarations.

  /Users/thomas/bothrs/open-source/packages/translations/src/index.ts:2:15
    1 | import { useState, useEffect } from 'react'
  > 2 | import i from 'i18next'
  >   |               ^^^^^^^^^^ Cannot find module 'i18next' or its corresponding type declarations.
    3 | import ChainedBackend from 'i18next-chained-backend'
    4 |

✨ Built in 734ms

dist/index.js      3.56 KB    33ms
dist/module.js     3.16 KB    18ms
dist/types.d.ts      255 B     7ms
```

--- a/packages/translations/package.json
+++ b/packages/translations/package.json
@@ -44,6 +44,7 @@
   "devDependencies": {
     "@types/node": "^16.11.6",
     "@types/react": "^17.0.24",
+    "i18next": "^20.0.0",
     "parcel": "^2.0.0",
     "tslib": "^2.3.1",
     "typescript": "^4.4.2"

